### ISSUE TITLE
Sending messages as a server

### ISSUE THREAD

**Issue by hendersonhc** [dma-ais/AisLib#32]
In our use case we want to set up a server for sending AIS message to 1 or more clients.
The server obtains the AIS data from a simulator. The messages are converted in the AIS format ans transmitted to any connected client. The client can be an operational system.

There is one small example at the end of the readme file using the AisTcpReader that sends a message to a given client with a known host and port number. We couldn't make this example working but we also searching for a simple setup for our use case. Can you provide an example on how to do this ?

**Comment by bergtwvd** [comment]
It would be helpful to have an example on how to do this. The present https://github.com/dma-ais/AisLib#sending-a-message is rather limited.

**Comment by tbsalling** [comment]
I agree. This project is currently somewhat limited on resources, but we can keep this issue open until there's time to enhance the description. If, in the mean time, one of you get it to work, it would be a great contribution if you could send us a pull request with a better description.

**Comment by hendersonhc** [comment]
I trying to set up my communication using the AirBusLauncher example by changing it a little bit. It now only set up a communication without sending a message. I want to add the sending and receiving of a message.
For me the class names are very  confusing. There are 4 classes: TcpServerProvider, TcpClientProvider,  TcpServerConsumer and TcpWriterConsumer. Which class should I use for sending my AIS message and which class should I use for receiving the AIS message assumed that they are connected using the same port ?
And what are the methods that  I should use for sending and receiving ?

**Comment by tbsalling** [comment]
@kaspernielsen do you remember?
